src/coreComponents/physicsSolvers/fluidFlow/IsothermalCompositionalMultiphaseFVMKernels.hpp

```diff
@@ -1064,6 +1154,8 @@ class DirichletFaceBasedAssemblyKernel : public FaceBasedAssemblyKernel< NUM_COM
     : Base( numPhases,
             rankOffset,
             hasCapPressure,
+            0,      // no C1-PPU
+            0.0,    //
```

Comment:
And then here, once we have the `UpwindingScheme` enum in place, we will just pass `UpwindingScheme::PhasePotentialUpwinding` which will be cleaner